edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+  <h4> {% trans "See Also" %} </h4>
```

Comment:
Not sure what is intended here. @chrisndodge @pbaruah
I'd like to review with you both what info should be presented or linked, and whether to topics in the Learner's Guide or FAQ on edx.org.